funsor/terms.py

```diff
@@ -457,19 +474,12 @@ def sample(self, sampled_vars, sample_inputs=None, rng_key=None):
         if sampled_vars.isdisjoint(self.inputs):
             return self
 
-        result = instrument.debug_logged(self.unscaled_sample)(
+        result = instrument.debug_logged(self._sample)(
             sampled_vars, sample_inputs, rng_key
         )
-        if sample_inputs is not None:
-            log_scale = 0
-            for var, domain in sample_inputs.items():
-                if var in result.inputs and var not in self.inputs:
-                    log_scale -= math.log(domain.dtype)
-            if log_scale != 0:
-                result += log_scale
         return result
 
-    def unscaled_sample(self, sampled_vars, sample_inputs, rng_key=None):
+    def _sample(self, sampled_vars, sample_inputs, rng_key=None):
         """
         Internal method to draw an unscaled sample.
```

Comment:
Update this docstring to clarify the new difference between `sample` and `_sample`?